draft-opsarea-rfc5706bis.md

```diff
@@ -143,13 +153,9 @@ author:
    should avoid having manageability pushed for a later phase of the
    development of the standard.
 
-   In discussing the importance of considering operations and
-   management, this document sets forth a list of guidelines and a
-   checklist of questions to consider (see Appendix A), which a protocol
-   designer or reviewer can use to evaluate whether the protocol and
-   documentation address common operations and management needs.
-   Operations and management are highly dependent on their environment,
-   so most guidelines are subjective rather than objective.
+   A list of guidelines and a
+   checklist of questions to consider, which a reviewer can use to evaluate whether the protocol and
+   documentation address common operations and management needs, is provided in {{CHECKLIST}}.
 
```

Comment:
We may decide in the future if we live this mention or remove it completely.

Reply:
For now, it's probably good to leave it and perhaps a comment to review this checklist in the light of the final 5607bis text to ensure it is up-to-date.